### DISCUSSION TITLE
Star Sports Hong Kong/China

### DISCUSSION THREAD

**Discussion by thedarkdragon11** [iptv-org/community#776] [Q&A]
### Channel Name

Star Sports (East Asian TV channel)

### Channel ID

StarSports.cn

### Notes

There used to be a stream for this but now offline.

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by thedarkdragon11** [comment]
Update: There's a stream here: http://live.tvfix.org/play/starsports but m3u8 link doesn't work as is.

**Comment by freearhey** [comment]
According to our database this channel was closed on 1 October 2021.

https://iptv-org.github.io/channels/kr/StarSports
